browser/search.mjs

```diff
@@ -103,24 +113,26 @@ const search = (state, push) => {
   })
 
   const searchTokenItem = (token, index, tokens) => {
-    console.log(token, index, tokens)
     const count = issuesPerToken[token] || 0
 
     if (token === 'or') {
-      return html`<li class="issue-search-query-or-item">or</li>`
+      return html`<li class="">or</li>`
```

Comment:
is the lack of class here intentional? why not `<li>or</li>` in that case?

Reply:
The class is no longer needed because it needs no styling. I'll remove the class.